src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -483,10 +480,25 @@ private bool ResourceExists(StopWordsRemovingEstimator.Language lang)
     }
 
     /// <summary>
-    /// Stopword remover removes language-specific list of stop words (most common words)
-    /// This is usually applied after tokenizing text, so it compares individual tokens
-    /// (case-insensitive comparison) to the stopwords.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="CustomStopWordsRemovingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of [Text]<xref:System.ReadOnlyMemory{System.Char}> |
```

Comment:
>Vector [](start = 35, length = 6)

should this be known-sized?

Reply:
No, it can be either known-sized or unknown size.
output would always be unknown size

---
In reply to: [276871092](https://github.com/dotnet/machinelearning/pull/3413#discussion_r276871092) [](ancestors = 276871092)